src/renderer/features/multisig-wallet-create/multisig-wallet/model/chain-model.ts

```diff
@@ -46,7 +46,11 @@ const calculateChainFeeFx = createEffect(
         signatories: [TEST_ACCOUNTS[0], TEST_ACCOUNTS[1]],
       });
 
-      const fee = await transactionService.getTransactionFee(fakeTx, api);
+      const fee = await withTimeout(transactionService.getTransactionFee(fakeTx, api), 5000, null);
```

Comment:
Fee should be calculated if chain is active and connected. Timeout works really bad in case of fees calc because usually it's required data for user